### ISSUE TITLE
Story

### ISSUE THREAD

**Issue by BinaryBrain** [BinaryBrain/Elodie-Dream#30]
It would be cool to write a little story to put at the beginning of each levels.

**Comment by Protectator** [comment]
"a little story to put" ?
Unless there's a real story growing each level, I don't think this really adds anything interesting to the game, but I'm not against it if someone wants to do it.

**Comment by BinaryBrain** [comment]
Currently, people don't understand what is a Poro if they didn't play LoL. Then, there are like "Wow, cool, I'm running after them, and... that's all". And everytime we change the environment, it's "OMG! I'm in a volcano! wait... why am I here?".
I think we don't need a big scenario. Just a cool justificative little story.
But it's only what I think. If nobody's agree with that, then it's ok, I guess.

**Comment by Protectator** [comment]
I still think we explained all of this by saying she is in a dream, and she has to go to through multiple environments to reach the Poro Island.

**Comment by jrabasco** [comment]
I don't think it is a must-have at all, we already explain enough and we should focus on other stuff.

**Comment by BinaryBrain** [comment]
OK, then, I'll remove it from the milestone!
